### PR TITLE
Add nodeID to heartbeat debug log

### PR DESCRIPTION
**Pull request by jkendzorra**
Useful to correlate TCP connections on the proxy node to actual nodes

### CODE REVIEW

File: lib/reversetunnel/localsite.go [hunk 1]
```diff
@@ -376,9 +376,9 @@ func (s *localSite) handleHeartbeat(rconn *remoteConn, ch ssh.Channel, reqC <-ch
 				}
 			}
 			if roundtrip != 0 {
-				s.log.WithFields(log.Fields{"latency": roundtrip}).Debugf("Ping <- %v.", rconn.conn.RemoteAddr())
+				s.log.WithFields(log.Fields{"latency": roundtrip}).Debugf("Ping <- %v (nodeID: %v)", rconn.conn.RemoteAddr(), rconn.nodeID)
```
Comment (awly) [inline lib/reversetunnel/localsite.go:379]:
```suggestion
				s.log.WithFields(log.Fields{"latency": roundtrip}).Debugf("Ping <- %v (nodeID: %q)", rconn.conn.RemoteAddr(), rconn.nodeID)
```

File: lib/reversetunnel/localsite.go [hunk 2]
```diff
@@ -376,9 +376,9 @@ func (s *localSite) handleHeartbeat(rconn *remoteConn, ch ssh.Channel, reqC <-ch
...
 			} else {
-				log.Debugf("Ping <- %v.", rconn.conn.RemoteAddr())
+				log.Debugf("Ping <- %v (nodeID: %v)", rconn.conn.RemoteAddr(), rconn.nodeID)
```
Comment (awly) [inline lib/reversetunnel/localsite.go:381]:
```suggestion
				log.Debugf("Ping <- %v (nodeID: %q)", rconn.conn.RemoteAddr(), rconn.nodeID)
```

File: lib/reversetunnel/remotesite.go [hunk 3]
```diff
@@ -364,9 +364,9 @@ func (s *remoteSite) handleHeartbeat(conn *remoteConn, ch ssh.Channel, reqC <-ch
 				}
 			}
 			if roundtrip != 0 {
-				s.WithFields(log.Fields{"latency": roundtrip}).Debugf("Ping <- %v.", conn.conn.RemoteAddr())
+				s.WithFields(log.Fields{"latency": roundtrip}).Debugf("Ping <- %v (nodeID: %v)", conn.conn.RemoteAddr(), conn.nodeID)
```
Comment (awly) [inline lib/reversetunnel/remotesite.go:367]:
```suggestion
				s.WithFields(log.Fields{"latency": roundtrip}).Debugf("Ping <- %v (nodeID: %q)", conn.conn.RemoteAddr(), conn.nodeID)
```

File: lib/reversetunnel/remotesite.go [hunk 4]
```diff
@@ -364,9 +364,9 @@ func (s *remoteSite) handleHeartbeat(conn *remoteConn, ch ssh.Channel, reqC <-ch
...
 			} else {
-				s.Debugf("Ping <- %v.", conn.conn.RemoteAddr())
+				s.Debugf("Ping <- %v (nodeID: %v)", conn.conn.RemoteAddr(), conn.nodeID)
```
Comment (awly) [inline lib/reversetunnel/remotesite.go:369]:
```suggestion
				s.Debugf("Ping <- %v (nodeID: %q)", conn.conn.RemoteAddr(), conn.nodeID)
```

**Review by awly — COMMENTED** [review]
ok to test

**Comment by awly** [comment]
LGTM, just some formatting tweaks to make output less confusing when `nodeID` is empty

**Review by awly — APPROVED** [review]

**Comment by klizhentas** [comment]
hey folks, why not use log.Fields structured event to add nodeID instead of adding to the string here?

**Review by awly — APPROVED** [review]

**Comment by awly** [comment]
retest this please

**Pull request merged**